<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogQueryStatistics</class>
 <widget class="QDialog" name="DialogQueryStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询统计</string>
  </property>
  <widget class="QGroupBox" name="groupBoxQueryInput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei UI</family>
    </font>
   </property>
   <property name="title">
    <string>统计时间输入</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButtonRecent">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
         <property name="text">
          <string>近期</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
         <property name="text">
          <string>近</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPeriodCount">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxPeriodType">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButtonPrevious">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
         <property name="text">
          <string>往期</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEditDateStart">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2019</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
         <property name="text">
          <string>至</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEditDateEnd">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2019</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxQueryOutput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei UI</family>
    </font>
   </property>
   <property name="title">
    <string>统计结果</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
         <property name="text">
          <string>总计停放</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditVehicleTotalCount">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
         <property name="text">
          <string>辆，总计收费</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditVehicleTotalCharge">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
          </font>
         </property>
         <property name="text">
          <string>元</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonQuery">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrint">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打印记录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
